api/features/feature_external_resources/models.py

```diff
@@ -79,9 +80,19 @@ def execute_after_save_actions(self):
             .get(id=self.feature.project.organisation_id)
             .github_config.first()
         ):
+            if self.type == "GITHUB_PR":
+                pattern = r"github.com/([^/]+)/([^/]+)/pull/\d+$"
+            elif self.type == "GITHUB_ISSUE":
+                pattern = r"github.com/([^/]+)/([^/]+)/issues/\d+$"
```

Comment:
Maybe to future proof this it could be helpful to add an `else` keyword and raise an exception.

Reply:
I don't think it's necessary to implement this, because I am already validating it before creating the feature external resource here:

https://github.com/Flagsmith/flagsmith/blob/135711662f938fa7a2518ac2f7a1c70ac75b0cc3/api/features/feature_external_resources/views.py#L79